crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -250,36 +277,14 @@ pub(crate) trait StateWriteExt: StateWrite {
         Ok(())
     }
 
-    #[instrument(
-        skip_all,
-        fields(address = %address.display_address(),
-        withdrawal_event_id, block_num),
-        err(level = Level::DEBUG)
-    )]
-    async fn check_and_set_withdrawal_event_block_for_bridge_account<T: AddressBytes>(
+    #[instrument(skip_all)]
+    fn put_withdrawal_event_block_for_bridge_account<T: AddressBytes>(
```

Comment:
```suggestion
    fn put_withdrawal_event_block_height_for_bridge_account<T: AddressBytes>(
```

Reply:
Done in [73ec272](https://github.com/astriaorg/astria/pull/2142/commits/73ec27210915e8a170768ff722a20d45297b4f81) (renamed to match suggestion for associated getter).